cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,52 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import SignInPageObject from "../support/pages/signIn.pageObject";
+const signInPage = new SignInPageObject();
+import ArticlePageObject from "../support/pages/article.pageObject";
+const articlePage = new ArticlePageObject();
 describe('Article', () => {
-  before(() => {
-
-  });
-
-  beforeEach(() => {
-    cy.task('db:clear');
-  });
+ beforeEach(() => {
+   let user;
+   cy.task('db:clear');
+   cy.task('generateUser').then((generateUser) => {
+      user = new Object(generateUser);
```

Comment:
The use of `new Object(generateUser)` is unnecessary. You can directly assign `user = generateUser;` since `generateUser` is already an object.